R/panel.r

```diff
@@ -27,7 +27,7 @@ new_panel <- function() {
 # @param plot_data the default data frame
 # @return an updated panel object
 train_layout <- function(panel, facet, data, plot_data) {
```

Comment:
panel.R should be deleted - I think I've grabbed everything I needed from there but was holding off for a bit...